src/test/java/org/folio/api/AllowedServicePointsApiTest.java

```diff
@@ -334,6 +336,60 @@ void replaceFailsWhenEcsTlrIsNotFound() {
       .expectStatus().isEqualTo(500);
   }
 
+  @Test
+  void allowedSpWithItemLevelReturnsResultSpInResponsesFromDataTenant() {
+    var searchItemResponse = new SearchItemResponse();
+    searchItemResponse.setTenantId(TENANT_ID_COLLEGE);
+    searchItemResponse.setInstanceId(INSTANCE_ID);
+    searchItemResponse.setId(ITEM_ID);
+
+    wireMockServer.stubFor(get(urlMatching(SEARCH_ITEM_URL))
+      .withHeader(HEADER_TENANT, equalTo(TENANT_ID_CONSORTIUM))
+      .willReturn(jsonResponse(asJsonString(searchItemResponse), SC_OK)));
+
+    var allowedSpResponseConsortium = new AllowedServicePointsResponse();
+    allowedSpResponseConsortium.setHold(Set.of(
+      buildAllowedServicePoint("SP_consortium_1"),
+      buildAllowedServicePoint("SP_consortium_2")));
+    allowedSpResponseConsortium.setPage(null);
+    allowedSpResponseConsortium.setRecall(Set.of(
+      buildAllowedServicePoint("SP_consortium_3")));
+
+    var allowedSpResponseCollege = new AllowedServicePointsResponse();
+    allowedSpResponseCollege.setHold(Set.of(
+      buildAllowedServicePoint("SP_college_1")));
+    allowedSpResponseCollege.setPage(null);
+    allowedSpResponseCollege.setRecall(Set.of(
+      buildAllowedServicePoint("SP_college_2")));
+
+    User requester = new User().patronGroup(PATRON_GROUP_ID);
+    wireMockServer.stubFor(get(urlMatching(USER_URL))
+      .withHeader(HEADER_TENANT, equalTo(TENANT_ID_CONSORTIUM))
+      .willReturn(jsonResponse(asJsonString(requester), SC_OK)));
+
+    wireMockServer.stubFor(get(urlMatching(ALLOWED_SERVICE_POINTS_MOD_CIRCULATION_URL_PATTERN))
+      .withHeader(HEADER_TENANT, equalTo(TENANT_ID_CONSORTIUM))
+      .willReturn(jsonResponse(asJsonString(allowedSpResponseConsortium), SC_OK)));
+
+    wireMockServer.stubFor(get(urlMatching(ALLOWED_SERVICE_POINTS_MOD_CIRCULATION_URL_PATTERN))
+      .withHeader(HEADER_TENANT, equalTo(TENANT_ID_COLLEGE))
+      .willReturn(jsonResponse(asJsonString(allowedSpResponseCollege),
+        SC_OK)));
+
+    doGet(
+      ALLOWED_SERVICE_POINTS_URL + format("?operation=create&requesterId=%s&itemId=%s",
+        REQUESTER_ID, ITEM_ID))
+      .expectStatus().isEqualTo(200)
+      .expectBody().json(asJsonString(allowedSpResponseConsortium));
+
+    wireMockServer.verify(getRequestedFor(urlMatching(
+      ALLOWED_SERVICE_POINTS_MOD_CIRCULATION_URL_PATTERN))
```

Comment:
let's verify that the call was made to the correct tenant. let's verify the second call as well.